examples/docker-compose/README.md

```diff
@@ -0,0 +1,85 @@
+This folder contains files and sub-folders that demonstrate how to run docker-stack notebook containers using Docker Compose on a Docker Machine-controlled host.
+
+## Pre-requisites
+
+* [Docker Engine](https://docs.docker.com/engine/) 1.10.0+
+* [Docker Machine](https://docs.docker.com/machine/) 0.6.0+
+* [Docker Compose](https://docs.docker.com/compose/) 1.6.0+
+
+See the [installation instructions](https://docs.docker.com/engine/installation/) for your environment.
+
+## Provision Docker machines
```

Comment:
Maybe add a note here about how this will work on **any** docker-machine supported host. The two below are just examples. Scripts welcome as PRs!